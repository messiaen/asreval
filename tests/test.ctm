;;
;; Comments follow ';;'
;;
;; The Blank lines are ignored

;;
7654 A 11.34 0.2 YES -6.763
7654 A 12.00 0.34 YOU -12.384530
7654 A 13.30 0.5 CAN 2.806418
7654 A 17.50 0.2 AS 0.537922
7654 B 1.34 0.2 I -6.763
7654 B 2.00 0.34 CAN -12.384530
7654 B 3.40 0.5 ADD 2.806418
7654 B 7.00 0.2 AS 0.537922
7655 A 11.34 0.2 YES -6.763
7655 A 12.00 0.34 YOU -12.384530
7655 A 13.30 0.5 CAN 2.806418
7655 A 17.50 0.2 AS 0.537922
7655 B 1.34 0.2 I -6.763
7655 B 2.00 0.34 CAN -12.384530
7655 B 3.40 0.5 ADD 2.806418
7655 B 7.00 0.2 AS 0.537922
7677 A 0.00 1.0 HERE
7677 A 1.00 1.0 IS
7677 A 2.00 1.0 A
7677 A 3.00 1.0 TEST
7677 A 4.00 1.0 UTTERANCE

