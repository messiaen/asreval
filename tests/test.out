[CnetEdge(start_node=0, end_node=5, start_time=0.0, end_time=0.55, word='THE', score=4.5302e-09),
CnetEdge(start_node=15, end_node=21, start_time=1.27, end_time=1.53, word='THE', score=3.65025e-07),
CnetEdge(start_node=18, end_node=21, start_time=1.35, end_time=1.53, word='THE', score=8.20543e-07),
CnetEdge(start_node=20, end_node=21, start_time=1.27, end_time=1.53, word='THE', score=3.65025e-07),
CnetEdge(start_node=23, end_node=26, start_time=2.74, end_time=2.86, word='THE', score=3.18141e-05),
CnetEdge(start_node=24, end_node=26, start_time=2.77, end_time=2.86, word='THE', score=6.80621e-05),
CnetEdge(start_node=25, end_node=26, start_time=2.76, end_time=2.86, word='THE', score=5.83629e-05),
CnetEdge(start_node=28, end_node=33, start_time=2.8, end_time=2.86, word='THE', score=0.000249758),
CnetEdge(start_node=29, end_node=33, start_time=2.79, end_time=2.86, word='THE', score=0.000312958),
CnetEdge(start_node=30, end_node=33, start_time=2.78, end_time=2.86, word='THE', score=0.00026477),
CnetEdge(start_node=31, end_node=33, start_time=2.78, end_time=2.86, word='THE', score=0.000286845),
CnetEdge(start_node=32, end_node=33, start_time=2.78, end_time=2.86, word='THE', score=0.000320936),
]